third_party/move/move-compiler-v2/tests/visibility-checker/call_friend_only.exp

```diff
@@ -1,13 +1,19 @@
 
 Diagnostics:
-error: `public(friend)` function `0xdeadbeef::M::id` cannot be called from `0xdeadbeef::Q::calls_bar` because `0xdeadbeef::Q` is not a friend of `0xdeadbeef::M`
-   ┌─ tests/visibility-checker/call_friend_only.move:46:9
+error: function `0xdeadbeef::M::bar` cannot be called from function `0xdeadbeef::Q::calls_bar` because module `0xdeadbeef::Q` is not a `friend` of `0xdeadbeef::M`
```

Comment:
Maybe we should also move all these tests out from here and into `tests/checking` to keep up with the existing convention?